configure.ac

```diff
@@ -71,6 +71,12 @@ case $host in
   ;;
 esac
 
+AC_ARG_WITH([seccomp],
+  [AS_HELP_STRING([--with-seccomp],
+  [enable experimental syscall sandbox feature (default is yes if seccomp-bpf is detected under Linux x86_64)])],
+  [use_syscall_sandbox=$withval],
```

Comment:
For a little more legibility here would suggest

- Replacing current uses of `use_syscall_sandbox` with `seccomp_found` in this file
- Before the `AM_CONDITIONAL`, adding
```autoconf
dnl Currently only enable -sandbox=<mode> feature if seccomp is found.
dnl In the future, sandboxing could be also be supported with other
dnl sandboxing mechanisms besides seccomp
use_syscall_sandbox=$seccomp_found
```

Reply:
Good idea! Fixed! :)